### ISSUE TITLE
Homepage Demo Video

### ISSUE THREAD

**Issue by joepavitt** [flowforge/website#202]
### Description

For 1.0 Release, we are revamping the website to ensure clear messaging around what FlowForge is and offers as a product. As part of this, we need a 1-2 minute video demo'ing FlowForge and it's capabilities.

<img width="1155" alt="Screenshot 2022-10-17 at 15 56 42" src="https://user-images.githubusercontent.com/99246719/196211045-61b62f2d-b734-48fd-a542-2709414cef7d.png">

Open Questions for this video:
- What do we demo?
    - Focus on FF already being installed (or demo'ing FF Managed)
    - Show how easy it is to setup a project and build/deploy a flow to a device
        - Thinking showing a device deployment is better as covers more functionality, but still shows the Project > Snapshot > Deploy pipeline

**Comment by robmarcer** [comment]
@joepavitt as discussed I think this would really benefit from Nick being on the video as a talking head.

**Comment by joepavitt** [comment]
Yep - agreed. @knolleary will assign to you, as we've discussed this too. @robmarcer can you share the draft story you recorded please?

**Comment by joepavitt** [comment]
Alternative Design. Think I prefer my original plan.

<img width="1093" alt="Screenshot 2022-10-20 at 15 25 12" src="https://user-images.githubusercontent.com/99246719/196975991-72f65e6c-dcae-465a-853e-7533672eb729.png">

**Comment by knolleary** [comment]
My plan was to start from the point of a user who is already signed up and a member of a team - so we can get to the 'create a node-red project easily' bit much quicker.

1. Login
2. Create project
3. Open Node-RED - create a simple flow that includes logging regularly to console.
4. In platform show other views of the project, include logs that should the flow console output
5. Register a device, download its credentials file, assign it to the project
6. In terminal - run device agent
7. In platform, create a snapshot, set as target
8. In terminal - show the flow console output

Need a good wrap up as well.

**Comment by joepavitt** [comment]
wrap up should be clear CTAs, something like:

> "If you want to play with FlowForge for free, you can head to Github and install our open-source version. Alternatively, you can head to our own managed instance (linked in the description) and get started straight away."

**Comment by joepavitt** [comment]
I am cautious of showing off console/terminal for a platform that is meant to be low-code - can we keep this browser-constrained?

**Comment by joepavitt** [comment]
Target time of 2 mins (max)
